docs/src/main/paradox/index.md

```diff
@@ -8,6 +8,7 @@ The Akka Thread Starvation Detector is a diagnostic tool that monitors the dispa
 @@@ index
```

Comment:
We should also have a summary of the Config Checker above. Now it only mentions Starvation Detector.

Reply:
added